model/src/wav_shr_flags.F90

```diff
@@ -0,0 +1,1201 @@
+!> @file wav_shr_flags
```

Comment:
This seems related to if-def removals and not this PR, but perhaps I've missed something?  Is this file necessary for this PR?

Reply:
We don't need all the flags, but it was simply easier to keep the original list together.

Reply:
If we backed out the W3_SBS change, can all of this be backed out?